web/src/components/DatePicker.tsx

```diff
@@ -95,71 +57,49 @@ const CustomDatePicker = ({
   calendarLabel,
 }: CustomDatePickerProps): ReactElement => {
   const { t } = useTranslation('events')
-  const [isCalendarOpen, setIsCalendarOpen] = useState(false)
   const [datePickerError, setDatePickerError] = useState('')
-  const [value, setValue] = useState('')
-  const maxInputLength = 10
+  const { languageCode } = useCityContentParams()
+  const isError = error || datePickerError
 
-  const handleDateChange = (date: Date | null) => {
-    if (isValidJsDate(date)) {
-      setDate(DateTime.fromJSDate(date ?? new Date()))
-    } else if (value.length < maxInputLength) {
-      setDatePickerError(t('invalidDate'))
-    }
-  }
-  const handleDateError = (date: string) => {
-    if (!containsOnlyDigits(date.replace('.', '')) || date.length > maxInputLength) {
-      return
-    }
-    setValue(date)
-    const errorString = 'unit out of range'
-    setDatePickerError('')
-    try {
-      DateTime.fromFormat(date, 'dd.MM.yyyy')
-    } catch (e) {
-      if (e instanceof Error && String(e.message).includes(errorString)) {
-        setDatePickerError(t('outOfRangeError'))
-      }
+  const handleDateChange = (newValue: DateTime | null) => {
+    if (newValue) {
+      setDate(newValue)
+      setDatePickerError('')
     }
   }
 
   useEffect(() => {
     if (date == null) {
-      setValue('')
       setDatePickerError('')
     }
   }, [date])
 
   return (
     <DateContainer>
-      <StyledInputWrapper>
-        <StyledInput
-          open={isCalendarOpen}
-          value={value}
-          popperPlacement='bottom'
-          selected={isValidJsDate(date?.toJSDate() ?? null) ? date?.toJSDate() : null}
-          onSelect={e => {
-            setValue(DateTime.fromJSDate(e ?? new Date()).toFormat('dd.MM.yyyy'))
-            setDatePickerError('')
+      <LocalizationProvider dateAdapter={CustomAdapterLuxon} adapterLocale={languageCode}>
```

Comment:
❓ Does it make sense to wrap every date picker separately in this `LocalizationProvider` instead of adding this around the whole app?

Reply:
Oh this is a provider already so I think yes. I will move it to `app.tsx`